src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Google Inc.
```

Comment:
Please add tests for ProfileServlet controller

Reply:
I haven't been able to add  the username to the URL but I added some small tests on the ProfileServlet